tutor/src/components/book-page.cjsx

```diff
@@ -6,7 +6,7 @@ classnames = require 'classnames'
 
 {BookContentMixin} = require './book-content-mixin'
 
-{default: {BookPageExerciseShell}} = require './book-page/exercise'
+{default: {ReferenceBookExerciseShell}} = require './book-page/exercise'
 RelatedContent = require './related-content'
```

Comment:
That's a strange way to require a named export, I've usually seen that as:
```
{ReferenceBookExerciseShell} = require './book-page/exercise'
```

Maybe there's a reason for doing it that way though?

Reply:
oh yeah! 👍